Makefile

```diff
@@ -25,11 +27,11 @@ verify:
 
 .PHONY: memcheck_tests
 memcheck_tests:
-	cuda-memcheck --leak-check full --error-exitcode 1 pytest $(PYTEST_CI_ARGS) tests/
+	compute-sanitizer --leak-check full --error-exitcode 1 pytest -m $(MEMCHECK_MARKER) $(PYTEST_CI_ARGS)
```

Comment:
would it be worth introducing and reverting a memory leak in this branch to make sure we're still catching them?

Reply:
Good catch, looks like this isn't working in gitlab at the moment

A bit more information here:
https://github.com/NVIDIA/nvidia-docker/issues/454